dev/reference/mysql-compatibility.md

```diff
@@ -149,3 +149,23 @@ TiDB supports **all of the SQL modes** from MySQL 5.7 with minor exceptions:
         - For MySQL 5.7: `OFF`
         - For MySQL 8.0: `ON`
     
+### Date and Time
+
+#### Time zone
+
+By default, MySQL time zone is set to the server's time zone and calculated with the system time zone rules such as the beginning of daylight saving time. Befor loading [time zone information table](https://dev.mysql.com/doc/refman/8.0/en/time-zone-support.html#time-zone-installation), you cannot specify the time zone with the time zone name. 
+
+All the time zone names are available in TiDB without loading the time zone information table. The system calculates based on its own time zone rules which is generally the `tzdata` package. The calculation rules cannot be modified by loading time zone information table. 
```

Comment:
TiDB does the date time calculation according to the system time zone rules ... which will not be modified even if time zone information table is manually loaded in the MySQL way.